src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/server/HelidonServer.java

```diff
@@ -149,5 +154,11 @@ public void adaptMavenModel(Model pomFile, JessieModel model, boolean mainProjec
         pomFile.addProperty("helidonVersion", helidonVersion);
         pomFile.addProperty("mpVersion", mpVersion);
 
+        Parent parent = new Parent();
```

Comment:
When selecting MP 2.2, I get the following error when I perform `mvn clean package`on the generated project.

`[ERROR]     Non-resolvable parent POM for com.example:helidon1-service-a:1.0-SNAPSHOT: Could not find artifact io.helidon.applications:helidon-mp:pom:1.2.0 in central (https://repo.maven.apache.org/maven2) and 'parent.relativePath' points at no local POM @ line 5, column 11 -> [Help 2]`

I assume the parent is only valid for Helidon2.

Reply:
You're right. I'll have a fix shortly. Thanks for the catch.

Reply:
I've updated the PR. The latest commit has git sign-off but GitHub check validation is claiming that it does not.